operator/gitops/argocd/pipeline-service/openshift-pipelines/tekton-config.yaml

```diff
@@ -87,6 +87,14 @@ spec:
     options:
       disabled: false
       configMaps:
+        config-observability:
+          data:
+            metrics.count.enable-reason: 'false'
```

Comment:
all the metrics props are defaults I have observed once we are deployed

I will run comparisons to confirm/deny whether tekton config application is an add/merge or a replace type of operation